test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... ok
test_to_degrees (__main__.TestExercice) ... FAIL
test_to_farenheit (__main__.TestExercice) ... ok
test_to_radians (__main__.TestExercice) ... ok

======================================================================
FAIL: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Imad\Documents\GitHub\2022H-ch3-ImadKhetir\test_exercice.py", line 83, in test_to_degrees
    self.assertListEqual(
AssertionError: Lists differ: [(114, 35, 29.612494192724625), (-230, 49, [77 chars]998)] != [(79.0980174562859, 5.881047377154118, 29.6[197 chars]998)]

First differing element 0:
(114, 35, 29.612494192724625)
(79.0980174562859, 5.881047377154118, 29.612494192724625)

- [(114, 35, 29.612494192724625),
-  (-230, 49, 0.7750116145507491),
-  (286, 28, 44.031235481811564),
-  (234, 20, 23.057550624127998)]
+ [(79.0980174562859, 5.881047377154118, 29.612494192724625),
+  (-278.1960349125718, 48.23790524569176, 0.7750116145507491),
+  (257.74504364071475, -15.297381557114704, 44.031235481811564),
+  (213.95544569810457, -2.673258113725865, 23.057550624127998)] : Mauvais calcul de radians -> degres

----------------------------------------------------------------------
Ran 7 tests in 0.005s

FAILED (failures=1)
